### DISCUSSION TITLE
What's the deal with backslashes inserted into markdown by Outline?

### DISCUSSION THREAD

**Discussion by ReessKennedy** [outline/outline#7046] [General] [answered]
Loving Outline but when I export I have to run a python script to clean the exports of all the backslashes and it sometimes doesn't work perfectly and the the wrong bits gets removed (i.e. bits of actual content I need to preserve). 

Ran a search in this forum for info on backslashes and can't find anything. I don't think it's part of normla markdown but I also know the project isn't using strict markdown. 

Still, if they are actually unnecessary for any critical function or if there is a way around this from better break recognition within the app it would have a few great benefits: 
1. Smaller database and data storage sizes (just the elimination of unnecessary characters)
2. Cleaner, more reliable and compatible exports (just eliminate need to scrub junk characters)

Penny for your thoughts? Sorry if I am missing some note that explains this. 

Example: 
<img width="1133" alt="Screenshot 2024-06-15 at 12 25 54 PM" src="https://github.com/outline/outline/assets/3970267/e42223f0-39b2-4b48-812c-81904e78f597">

**Comment by tommoor** [comment, marked as answer]
Until version 0.77.0 the slashes are required to represent multiple line breaks.

Now we've moved away from Markdown as canonical store, these can soon be removed.

They should already be stripped when exporting as markdown on an individual doc
